composer.json

```diff
@@ -11,12 +11,9 @@
     "require": {
         "php": "^7.4 || ^8.0",
```

Comment:
Can we drop `^7.4` support?

Reply:
`"dealerdirect/phpcodesniffer-composer-installer": "^1.0"` - `"php": ">=5.4"`
`"slevomat/coding-standard": "^8.15.0"` - `"php": "^7.2 || ^8.0"`
`"squizlabs/php_codesniffer": "^3.10"` - `"php": ">=5.4.0"`
`"webimpress/coding-standard": "^1.3"` - `"php": "^7.3 || ^8.0"`

I think we can leave it as it is.